alphascreener/cli.py

```
@@ -310,6 +310,17 @@ def evolve_review_last(days: int) -> None:
     except Exception as exc:
+        from sqlalchemy.exc import OperationalError
+
+        err_msg = str(exc)
+        if isinstance(exc, OperationalError) and ("no such table" in err_msg or "no such column" in err_msg):
+            click.echo(
+                "Error: Database schema not ready. Run:\n"
+                "  alembic upgrade head\n"
+                "to create/migrate the database tables, then retry.",
+                err=True,
+            )
+            raise SystemExit(1)
```

Ruff (0.15.13)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)